matminer/data_retrieval/retrieve_MDF.py

```diff
@@ -35,76 +34,67 @@ def __init__(self, anonymous=False, **kwargs):
-                      exclude_fields=None, match_ranges=None,
-                      exclude_ranges=None, unwind_arrays=True):
+    def api_link(self):
+        return "https://materialsdatafacility.org/"
+
+    def get_dataframe(self, criteria, properties=None, unwind_arrays=True):
         """
-        Retrieves data from the MDF API and formats it as
-        a Pandas Dataframe
+        Retrieves data from the MDF API and formats it as a Pandas Dataframe
 
         Args:
-            source_names ([str]): source names to include, e. g. ["oqmd"]
-            elements ([str]): elements to include, e. g. ["Ag", "Si"]
-            titles ([str]): titles to include, e. g. ["Coarsening of a semisolid
-                Al-Cu alloy"]
-            tags ([str]): tags to include, e. g. ["outcar"]
-            resource_types ([str]): resources to include, e. g. ["record"]
-            match_fields ({}): field-value mappings to include, e. g.
-                {"oqdm.converged": True}
-            exclude_fields ({}): field-value mappings to exclude, e. g.
-                {"oqdm.converged": False}
-            match_ranges ({}): field-range mappings to include, e. g.
-                {"oqdm.band_gap.value": [1, 5]}, use "*" for no lower
-                or upper bound, e. g. {"oqdm.band_gap.value": [1, "*"]},
-            exclude_ranges ({}): field-range mapping to exclude,
-                {"oqdm.band_gap.value": [3, "*"]} to exclude all
-                results with band gap higher than 3.
-            raw (bool): whether or not to return raw (non-dataframe)
-                output, defaults to False
+            criteria (dict): options for keys are
+                source_names ([str]): source names to include, e. g. ["oqmd"]
+                elements ([str]): elements to include, e. g. ["Ag", "Si"]
+                titles ([str]): titles to include, e. g. ["Coarsening of a
+                    semisolid Al-Cu alloy"]
+                tags ([str]): tags to include, e. g. ["outcar"]
+                resource_types ([str]): resources to include, e. g. ["record"]
+                match_fields ({}): field-value mappings to include, e. g.
+                    {"oqdm.converged": True}
+                exclude_fields ({}): field-value mappings to exclude, e. g.
+                    {"oqdm.converged": False}
+                match_ranges ({}): field-range mappings to include, e. g.
+                    {"oqdm.band_gap.value": [1, 5]}, use "*" for no lower
+                    or upper bound, e. g. {"oqdm.band_gap.value": [1, "*"]},
+                exclude_ranges ({}): field-range mapping to exclude,
+                    {"oqdm.band_gap.value": [3, "*"]} to exclude all
+                    results with band gap higher than 3.
+                raw (bool): whether or not to return raw (non-dataframe)
+                    output, defaults to False
             unwind_arrays (bool): whether or not to unwind arrays in
```

Comment:
need to document properties arg

Reply:
@albalu Also it appears that properties arg is not used anywhere in get_dataframe? Do we just want to select the properties columns at the end by changing to `return make_dataframe(results, unwind_arrays=unwind_arrays)[properties]`?